decidim-core/app/views/decidim/messaging/conversations/show.html.erb

```diff
@@ -7,3 +7,7 @@
 
   <%= render "reply", form: @form, conversation: conversation %>
 <% end %>
+
```

Comment:
It is advisable to put all javascript code inside the corresponding js files and not inside the views.
In this case it would be recommendable to perform the action in the js file when the desired html is fully loaded. Then, execute the method to scroll down to the bottom.